.grit/patterns/js/es6_exports.md

```diff
@@ -124,3 +125,24 @@ const { sub1, sub2 } = require('mod5'); // not handled
 export { sub1 };
 export { sub2 };
 ```
+
+### Transforms method definition shorthand
+
+```js
+const shorthand = 1;
+
+module.exports = {
+  shorthand,
+  fn(args) {
+    return 'impl';
+  },
+};
+```
+
+```js
+export const shorthand = 1;
+
+export const fn = function fn(args) {
+  return 'impl';
+};
```

Comment:
Is this actually valid? Aren't you redefining `fn` here?

Reply:
I think it's valid. From Safari console (because Chrome console allows to re-define `consts`):

<img width="387" alt="Screenshot 2024-02-05 at 11 08 13 AM" src="https://github.com/getgrit/stdlib/assets/456426/746cb785-94de-46c1-b7e9-fdd4261fe75c">


Tbh, my intention was `export function fn(args) {}` which I agree is more natural/concise, so I refactored to produce that.